document/CoreFormat.mdk

```diff
@@ -1023,36 +1031,36 @@ A `hash` object shall contain a property named `algorithm`, whose value is a str
 of the algorithm used to compute the hash value specified in the `value` property (&sect;[#hash-value]).
 This shall be one of the following:
 
-*	`BLAKE-256`
-*	`BLAKE-512`
-*	`ECOH`
-*	`FSB`
-*	`GOST`
-*	`Groestl`
-*	`HAS-160`
-*	`HAVAL`
-*	`JH`
-*	`MD2`
-*	`MD4`
-*	`MD5`
-*	`MD6`
-*	`RadioGatun`
-*	`RIPEMD`
-*	`RIPEMD-128`
-*	`RIPEMD-160`
-*	`RIPEMD-320`
-*	`SHA-1`
-*	`SHA-224`
-*	`SHA-256`
-*	`SHA-384`
-*	`SHA-512`
-*	`SHA-3`
-*	`Skein`
-*	`Snefru`
-*	`Spectral Hash`
-*	`SWIFFT`
-*	`Tiger`
-*	`Whirlpool`
+*	`"BLAKE-256"`
```

Comment:
Whoops, no, we cleaned up these values to be more in line with other properties. They are rendered as blake512, ecoh, fsb, etc. See JSON schema